Code/GraphMol/DetermineBonds/README.md

```diff
@@ -0,0 +1,51 @@
+# GSOC 2022: Integrating xyz2mol Into The RDKit 
+
+### Organization: OpenChemistry
+### Student: Sreya Gogineni
+### Mentors: Greg Landrum, Joey Storer, Jan H. Jensen
+
+This summer, I worked on integrating 'xyz2mol' into the RDKit, an open source cheminformatics library. '[xyz2mol](https://github.com/jensengroup/xyz2mol)' was originally developed by Professor Jan H. Jensen's research group at the University of Copenhagen, based off of the work published in [this paper](https://onlinelibrary.wiley.com/doi/10.1002/bkcs.10334) (DOI: 10.1002/bkcs.10334). 
```

Comment:
Could we add the paper reference to the doc string?  I doubt anyone but us will see the README.